config/locales/en.yml

```diff
@@ -3119,184 +3147,204 @@ en:
           title: You said you were a homeowner in %{filing_year}.
       nj_homeowner_property_tax:
         edit:
-          helper_description: TODO FILL THIS OUT
-          helper_heading: Where do I find how much property taxes I paid?
-          label: Property taxes paid in %{filing_year}
-          title: Enter the amount of property taxes paid to %{municipality} for your main home in %{filing_year}.
+          helper_anchor_description_html: If you received an ANCHOR benefit, the benefit amount does not affect the amount to report on this line. (<a href="https://www.nj.gov/treasury/taxation/anchor/" target="_blank" rel="noopener nofollow">ANCHOR offers property tax relief to New Jersey residents who own or rent property in New Jersey as their main home and meet certain income limits.</a>)
+          helper_anchor_heading: What if I received ANCHOR or Homestead benefits?
+          helper_description: You can review your quarterly tax bills and add up the total paid to your municipality.
+          helper_heading: Where can I find property taxes paid?
+          label_html: "<strong>Enter property taxes paid for %{filing_year}</strong>"
+          title: Enter the amount of property taxes you paid on your main home where you lived in %{filing_year}.
       nj_homeowner_property_tax_worksheet:
         edit:
-          anchor_homestead_helper_description: If you received an ANCHOR Benefit, the benefit amount does not affect the amount to report on this line.
-          anchor_homestead_helper_heading: ANCHOR or Homestead benefits?
-          helper_1: Good news – we think you might qualify for a credit or deduction.
-          helper_2_html: The next step is to fill in this <a href="https://www.nj.gov/treasury/taxation/pdf/current/1040_instr_pages/1040i_32.pdf" target="_blank" rel="noopener nofollow">worksheet</a>.
-          helper_3: It will ask you about where you lived and for how long. If you moved, for example, it will have space for you to list every place where you lived.
-          helper_4: Once you fill in the worksheet, come back to this page.
-          label_html: "<b>Enter Line 4 on Worksheet G</b> </br> (Your share of total property taxes paid)"
-          title: Homeowner Property Tax Worksheet
-          where_to_find_helper_description: You can review your quarterly tax bills and add up the total paid to your municipality.
-          where_to_find_helper_heading: Where can I find property taxes paid?
+          description_html: "<p>The next step is to fill in this <a href=\"https://www.nj.gov/treasury/taxation/pdf/wrkshtg.pdf\" target=\"_blank\" rel=\"noopener nofollow\">worksheet</a>.</p> \n<p>Once you fill in the worksheet, come back to this page.</p>\n<p>The worksheet will ask you about where you lived and for how long. If you moved, for example, it will have space for you to list every place you lived.</p>\n"
```

Comment:
[dust] this could be another candidate for using `|` instead of needing to escape strings and write out the line breaks?